### PR TITLE
Small update on how we get the number of cpu cores

### PR DESCRIPTION
**Pull request by quantuumsnot**
Changed the code a bit for better readability and future improvements

### CODE REVIEW

File: init.php [hunk 1]
```diff
@@ -9,28 +9,26 @@
...
-      fgets($process);
-      $cores = intval(fgets($process));
-      pclose($process);
-    }
-  } else {
-    $process = popen('sysctl -a', 'rb');
-    if (false !== $process) {
-      $output = stream_get_contents($process);
-      preg_match('/hw.ncpu: (\d+)/', $output, $matches);
+function cpu_cores($cores = 1) {
```
Comment (raxbg) [inline init.php]: I don't think we need `$cores` as a function parameter.

**Review by raxbg — CHANGES_REQUESTED** [review]
Please remove the function parameter and I will merge the changes.

File: init.php (hunk 1 above)
Comment (quantuumsnot) [reply init.php]: Yes, I put it there to lower the LoC count and initialize it with a default value at the same time. Anyway it was a tough battle to run PHP and test the script on 64bit Windows XP. I was surprised to see that PHP (5.4.45 TS 32bit) runs it almost instantaneously while PHP 7.0.9 needed almost 2 seconds for that :\

Comment (raxbg) [reply init.php]: Honestly, I am impressed that you even bothered trying this on Windows :smile:

**Review by raxbg — APPROVED** [review]

File: init.php (hunk 1 above)
Comment (quantuumsnot) [reply init.php]: Ye, Chrome died two times due to low memory ... those virtual machines are hungry

**Pull request merged**